cmd/skupper/skupper.go

```diff
@@ -243,7 +258,8 @@ installation that can then be connected to other skupper installations`,
 	cmd.Flags().StringVarP(&routerCreateOpts.AuthMode, "console-auth", "", "", "Authentication mode for console(s). One of: 'openshift', 'internal', 'unsecured'")
 	cmd.Flags().StringVarP(&routerCreateOpts.User, "console-user", "", "", "Skupper console user. Valid only when --console-auth=internal")
 	cmd.Flags().StringVarP(&routerCreateOpts.Password, "console-password", "", "", "Skupper console user. Valid only when --console-auth=internal")
-	cmd.Flags().BoolVarP(&routerCreateOpts.ClusterLocal, "cluster-local", "", false, "Set up skupper to only accept connections from within the local cluster.")
+	cmd.Flags().BoolVarP(&ClusterLocal, "cluster-local", "", false, "Set up skupper to only accept connections from within the local cluster.")
+	cmd.Flags().StringVarP(&routerCreateOpts.Ingress, "ingress", "", "loadbalancer", "[loadbalancer|route|none].")
```

Comment:
iiuc from the notes, "the default is loadbalancer unless OpenShift in which case it is route"

Need to set default here so that checks against Ingress will do the right thing (such as creating routes on OpenShift)

Reply:
Aha!
* Well No, the intention (and what I think it is what we decided) is that "LoadBalancer" is the default. There is no more logic evaluating if routes are available or not... it just use what you say in the --ingress flag (and the default is just the default). And for backward compatibility, if you say --cluster-local -> this translates to "--ingress none".

* In line: https://github.com/skupperproject/skupper/pull/395/files#diff-57d99a8b313f5331b29bc0bb6fc79911009716b2aede40f5b67fed9ce7d8516bR739
I am doing a verification, of the "RouteClient" available for the "--ingress route" option.

*Only for tests is that I am using some discovery.

What do you think?

Reply:
iiuc from the notes, "the default is loadbalancer unless OpenShift in which case it is route" -> correct, you are right, I am wrong